templates/land-see/land-see.amp.html

```diff
@@ -42,10 +61,82 @@ <h1 class="pt3">Hello AMP!</h1>
+        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
       </p>
     </div>
 
+    {{#accordion}}
+    <div id="accordion-container" class="accordion center items-center">
+      <amp-accordion layout="container" disable-session-states class="amp-accordion">
+        {{#section}}
+        <section {{expanded}} class="{{classes}}">
+          <h3 class="max-width-4 mx-auto">{{title}}</h3>
+          <div class="accordion-content max-width-4 mx-auto">
+            <amp-list src="{{list-src}}" height="430" layout="fixed-height">
+              <template type="amp-mustache">
+                {{#observer}}
+                <amp-position-observer
+                  on="{{action}}"
+                  layout="nodisplay">
+                </amp-position-observer>
+                {{/observer}}
+                <ul class="list-reset m0 p0 center flex justify-center">
+                  <%#land-see-category%>
+                  <li class="<%post-classes%> post-item mx1">
+                    <a href="<%post-url%>" class="text-decoration-none">
+                      <%#image%>
+                        <amp-img
+                          class="post-image <%classes%>"
+                          alt="<%alt%>"
+                          media="<%media-query%>"
+                          width="<%width%>"
+                          height="<%height%>"
+                          layout="<%layout%>"
+                          src="<%src%>">
+                        </amp-img>
+                      <%/image%>
+                    </a>
+                    <a class="post-category text-decoration-none" href="<%post-category-url%>"><%post-category%></a>
+                    <h4 class="pt1"><a class="text-decoration-none" href="<%post-url%>"><%post-title%></a></h4>
+                    <p class="pt1 pb3"><%post-copy%></p>
+                  </li>
+                  <%/land-see-category%>
+                </ul>
+              </template>
+            </amp-list>
+          </div>
+        </section>
+        {{/section}}
+      </amp-accordion>
+    </div>
+    {{/accordion}}
+
+    {{#instagram-cta}}
+    <div class="cta-wrapper mx-auto">
```

Comment:
why do we need this wrapper? can't we make the a block (and give it a nice class name to set the max-width 
 or use bass - max-width-1/max-width-2 etc)and mx-auto?

Reply:
The template calls for different y-margins than I had available in Basecss, at different breakpoints. I created this wrapper to follow that behavior on top of the media calls I already was passing into the `amp-anim` components. Let me play with bass and see about simplifying.